examples/alt_credential_auth.py

```diff
@@ -0,0 +1,13 @@
+from oauth2client.service_account import ServiceAccountCredentials
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+
+GOOGLE_SCOPES = ['https://www.googleapis.com/auth/drive']
+creds = ServiceAccountCredentials.from_json_keyfile_name('service-key.json, GOOGLE_SCOPES)
```

Comment:
missing second single quote for 'service-key.json'

Reply:
```suggestion
creds = ServiceAccountCredentials.from_json_keyfile_name('service-key.json', GOOGLE_SCOPES)
```

Reply:
Thanks! Good call

Reply:
thank you! this made it easy to get rolling